File: trail1/학점 계산기/credit-calculator.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        double sum = 0;
        int n = sc.nextInt();
        
        for(int i = 0; i < n; i++){
            double num = sc.nextDouble();
            sum += num;
        }

        double avg = (double) sum / n;

        System.out.printf("%.1f\n", avg);
        if (avg >= 4.0) {
            System.out.print("Perfect\n");
        } else if (avg >= 3.0) {
            System.out.print("Good\n");
        } else {
            System.out.print("Poor\n");
        }
    }
}

File: trail1/학점 계산기/credit-calculator.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int n = 0;
    scanf("%d", &n);

    double *arr = malloc(sizeof(double) *n);
    double sum= 0;

    for(int i = 0; i < n; i++){
        scanf("%lf", &arr[i]);
        sum += arr[i];
    }

    double avg = sum / n;

    printf("%.1lf\n", avg);
    if (avg >= 4.0) {
        printf("Perfect\n");
    } else if (avg >= 3.0) {
        printf("Good\n");
    } else {
        printf("Poor\n");
    }

    free(arr);
    return 0;
}